sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/README.md

```diff
@@ -18,102 +18,28 @@ dotnet add package Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents
-
-If the `accessTokenAcceptedVersion` property is set to `2`:
-6. Set the `Issuer URL to "https://login.microsoftonline.com/{tenantId}/v2.0"
-7. Set an 'Allowed Audience' to the Application ID (`appId`)
-
-If the `accessTokenAcceptedVersion` property is set to `1` or `null`:
-6. Set the `Issuer URL to "https://sts.windows.net/{tenantId}/"
-7. Set an 'Allowed Audience' to the Application ID URI (also known as`identifierUri`). It should be in the format of`api://{azureFunctionAppName}.azurewebsites.net/{resourceApiAppId}` or `api://{FunctionAppFullyQualifiedDomainName}/{resourceApiAppId}` if using a [custom domain name](https://learn.microsoft.com/azure/dns/dns-custom-domain#:~:text=Azure%20Function%20App%201%20Navigate%20to%20Function%20App,Custom%20domain%20text%20field%20and%20select%20Validate.%20).
-
-By default, the Authentication event trigger will validate that Azure Function authentication integration is configured and it will check that the **client** in the token is set to `99045fe1-7639-4a75-9d4a-577b6ca3810f` (via the `azp` or `appid` claims in the token).
-
-If you want to test your API against some other client that is not Microsoft Entra authentication events service, like using Postman, you can configure an _optional_ application setting:
-
-* **AuthenticationEvents__CustomCallerAppId** - the guid of your desired client. If not provided, `99045fe1-7639-4a75-9d4a-577b6ca3810f` is assumed.
-
-#### Have the trigger validate the token
-
-In local environments or environments that aren't hosted in the Azure Function service, the trigger can do the token validation. Set the following application settings in the [local.settings.json](https://learn.microsoft.com/azure/azure-functions/functions-develop-local#local-settings-file) file:
-
-* **AuthenticationEvents__TenantId** - your tenant ID
-* **AuthenticationEvents__AudienceAppId** - the same value as "Allowed audience" in option 1.
-* **AuthenticationEvents__CustomCallerAppId** (_optional_) - the guid of your desired client. If not provided, `99045fe1-7639-4a75-9d4a-577b6ca3810f` is assumed.
-
-An example `local.settings.json` file:
-
-```json
-{
-  "IsEncrypted": false,
-  "Values": {
-    "AzureWebJobsStorage": "UseDevelopmentStorage=true",
-    "FUNCTIONS_WORKER_RUNTIME": "dotnet",
-    "AuthenticationEvents__TenantId": "8615397b-****-****-****-********06c8",
-    "AuthenticationEvents__AudienceAppId": "api://46f98993-****-****-****-********0038",
-    "AuthenticationEvents__CustomCallerAppId": "46f98993-****-****-****-********0038"
-  }
-}
-```
+There are different approaches to authenticating HTTP requests to your function app and validating the token which can be found [here](https://learn.microsoft.com/entra/identity-platform/custom-extension-tokenissuancestart-setup?tabs=visual-studio%2Cazure-portal&pivots=nuget-library#configure-authentication-for-your-azure-function).
```

Comment:
The issue with the recommended approach here is that it is done via the azure portal. This section is subject to be updated without our knowledge, therefore lead to inaccurate instructions. I would much rather leave it in the documentation that already exists for it

Reply:
You need to have something in this section that allows for a complete end-to-end scenario with the least amount of friction for a developer just starting out.

For example, the portal docs can stay external, but the portal has nothing to do with local settings.   That example seems like it may be quite useful here to a developer just starting out.

Reply:
I have added the local setting json in here to make it easier for the developer